src/%IPM/CLI.cls

```diff
@@ -66,279 +65,273 @@ ClassMethod %Help(ByRef pCommandInfo) [ Final ]
 			
 			Kill tOneCommandStruct
 			Merge tOneCommandStruct = tCommandStruct(tCommand)
-			Do ..%HelpForCommand(tCommand,.tOneCommandStruct,tVerbose,tMarkdown)
+			Do ..%HelpForCommand(tCommand,.tOneCommandStruct,tVerbose,tMarkdown,$Get(tCommandStruct(-1, "maxLength"),0))
 			
 			Write !
 		}
 		
 		If 'tVerbose {
-			Write !, "For more detail, run:", !?2, "help <command-name>", !, "or", !?2, "help -v"
+			Write !!, "For more detail, run:", !?2, "help <command-name>", !, "or", !?2, "help -v"
 		}
 	}
 	Write !
 }
 
-ClassMethod %HelpForCommand(pCommandName As %String, ByRef pCommandStruct, pDetailed As %Boolean = 0, pMarkdownFormat As %Boolean = 0)
+ClassMethod %HelpForCommand(pCommandName As %String, ByRef pCommandStruct, pDetailed As %Boolean = 0, pMarkdownFormat As %Boolean = 0, pMaxLength As %Integer = 0)
```

Comment:
really, all these changes really necessary?
and removing added features, also necessary?

Reply:
This class is just a help generation, there should not be many important changes from HS
But, you just completely replaced it, with no care of changes